### ISSUE TITLE
Latest build fails, due to api error for specified biz.aQute.bnd version

### ISSUE THREAD

**Issue by LucaNerlich** [adobe/aem-modernize-tools#29]
The pom hardcodes the version 2.4.0 for biz.aQute.bnd as a dependency for the maven-bundle-pluing. This crashes for the latest build. Removing this specific versioned dependency, the build works fine again.

**Comment by jpmat296** [comment]
Another solution that works for me is to force version for `maven-bundle-plugin`:

````diff
diff --git a/parent/pom.xml b/parent/pom.xml
index 14420db..cf46ab6 100644
--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -280,6 +280,7 @@
                 <plugin>
                     <groupId>org.apache.felix</groupId>
                     <artifactId>maven-bundle-plugin</artifactId>
+                    <version>3.0.1</version>
                     <extensions>true</extensions>
                     <dependencies>
                         <dependency>
````

**Comment by bstopp** [comment]
This is resolved in #63 as the build has moved to the `bnd-maven-plugin`
